can/interfaces/kvaser/argument_parser.py

```diff
@@ -1,3 +1,9 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+TODO: Where is this used? Is this used?
```

Comment:
Not used. Feel free to remove this file.

Reply:
It is now gone.